### PR TITLE
Update Physical Property Analysis

### PR DESCRIPTION
**Pull request by SimonBoothroyd**
## Description

This PR updates the analysis of the benchmarking against the physical property data. In particular:

* The results files have been upgraded to the latest evaluator file format.
* The plots have been refreshed for the parsley manuscript to be more concise and to remove the statistics insets. 
* Unused plots have been removed.
* A set of summary statistics have been added.

## Status

  - [x] Ready to go

### CODE REVIEW

File: physical_properties/analysis/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    
+    - ``1a803034-7366-4a0c-9636-8ce3470f1bb1``
+    - ``1e3999e0-59e8-41f2-991c-e7ec6c7c7bee``
+    - ``a8229131-40d7-47d5-88a7-5910ca0a5421``
+    - ``c6121e84-6082-4976-b21d-2faa32119080``
+    - ``9111b26a-a5e2-4b38-a2e8-d9c4cbbec88d``
+    - ``acd8823e-a58f-4290-ad98-9901e1f1ffce``
+    - ``5bb33791-c951-4384-8e68-762ee0e77a26``
+
+  where omitted from the analysis as they were not successfully esimtated for
```
Comment (j-wags) [inline physical_properties/analysis/README.md]:
```suggestion
were omitted from the analysis as they were not successfully estimtated for 
```

File: physical_properties/analysis/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- The ``raw_data`` directory contains the original experimental test set and the 
+  corresponding estimated data sets.
+- The ``raw_data_v2`` directory contains *exactly* the same data as the ``raw_data``
+  directory, however the files have been updated to use the file format of the latest 
+  available version of the ``openff-evaluator`` framework (``0.2.1``). This conversion
+  was performed by the ``upgrade_result_files.py`` script.
+- The ``plots`` directory contains scatter plots comparing the experimental and estimated 
+  data sets. These were generated by the ``analyse.py`` script.
+- The ``statistics`` directory contains summary tabled of statistics comparing the
```
Comment (j-wags) [inline physical_properties/analysis/README.md]:
```suggestion
- The ``statistics`` directory contains summary tables of statistics comparing the 
```

File: physical_properties/analysis/analyse.py [hunk 3]
```diff
@@ -1,204 +1,236 @@
...
+    "acd8823e-a58f-4290-ad98-9901e1f1ffce",
+]
 
-preferred_units = {
-    'Density': unit.kilogram / unit.meter ** 3,
-    'DielectricConstant': unit.dimensionless,
-    'EnthalpyOfVaporization': unit.kilojoule / unit.mole,
-    'EnthalpyOfMixing': unit.kilojoule / unit.mole,
-    'ExcessMolarVolume': unit.centimeter ** 3 / unit.mole
+PROPERTY_TYPE_TO_TITLE = {
```
Comment (j-wags) [inline physical_properties/analysis/analyse.py:31]:
```suggestion
# Define templates for LATEX output
PROPERTY_TYPE_TO_TITLE = {
```

File: physical_properties/analysis/analyse.py [hunk 4]
```diff
@@ -1,204 +1,236 @@
...
-from openforcefield.typing.engines.smirnoff import ForceField
-from openforcefield.utils import UndefinedStereochemistryError
+from openff.evaluator import properties
+from openff.evaluator.datasets import PhysicalPropertyDataSet
+
+seaborn.set_context("paper", font_scale=2.2)
 
-from propertyestimator import unit
-from propertyestimator.utils.serialization import TypedJSONDecoder
+pyplot.rc('text', usetex=True)
```
Comment (j-wags) [inline physical_properties/analysis/analyse.py]: This line adds a hard dependency on latex (not conda-installable), which prevented me from running the script. Removing it didn't seem to have a major effect. Maybe consider making it a soft dependency.

```suggestion
try:
    pyplot.rc('text', usetex=True)
except:
    pass
```

**Review by j-wags — APPROVED** [review]
Looks really great. This sort of thing reminds me that I need to start modernizing and using type hints. A few quick things to consider, none of which are blocking:

* It may help to update the top-level manifest. Right now the QM folder is mentioned, but not the phys prop
* To rerun I did `conda create -n evaluator -c conda-forge -c omnia -c omnia/label/beta openff-evaluator seaborn` Is this worth documenting? 
* One line adds a hard dependency on latex -- Maybe consider making it soft (code change suggested in comment)

**Comment by davidlmobley** [comment]
Thanks!

**Pull request merged**